tests/libs/datasets/manual_filter_test.py

```diff
@@ -154,3 +159,11 @@ def test_manual_filter_per_bucket_tag():
     )
 
     test_helpers.assert_dataset_like(ds_out, ds_expected)
+
+
+def test_region_overrides_transform():
+    aggregator = statistical_areas.CountyToCBSAAggregator.from_local_public_data()
+    transformed = manual_filter.transform_region_overrides(
+        json.load(open(data.REGION_OVERRIDES_JSON)), aggregator.cbsa_to_counties_region_map
+    )
+    pprint.pprint(transformed)
```

Comment:
Is this a placeholder test or intentionally meant to just be a smoke test?

Reply:
Just a smoke test. Renamed.